### ISSUE TITLE
Can't create Minio buckets 'mc: command not found'

### ISSUE THREAD

**Issue by Christophvh** [laravel/homestead#1442]
## Versions
  - Vagrant: 2.2.9
  - Provider: Virtualbox 6.1
  - Homestead: 10.9

## Host operating system
Mac OS Catalina 10.15

## Homestead.yaml
```yaml
ip: 192.168.10.10
memory: 4048
cpus: 2
provider: virtualbox
authorize: ~/.ssh/id_rsa.pub
keys:
    - ~/.ssh/id_rsa
folders:
    -
        map: /Users/my-name/Documents/Projects/my-project
        to: /home/vagrant/code
sites:
    -
        map: homestead.test
        to: /home/vagrant/code/public
databases:
    - homestead
    - testing
features:
    - 
        mongodb: true
    - 
        minio: true
buckets:
    - name: minio-test-bucket
      policy: public
ports: 
    - send: 8086 
      to: 8086
    - send: 9200
      to: 9200
      protocol: tcp
name: my-project
hostname: my-project
```

Error after running vagrant up or vagrant up --provision. The box is working fine, but minion is not accessible.

```
==> my-project: Running provisioner: Creating Minio Bucket: your-bucket (shell)...
    my-project: Running: script: Creating Minio Bucket: your-bucket
    my-project: /tmp/vagrant-shell: line 3: mc: command not found
    my-project: /tmp/vagrant-shell: line 4: mc: command not found
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.

```

**Comment by Christophvh** [comment]
To give some extra information. It seems that minion is correctly installed, but i cant seem to start it

Running: `sudo systemctl status minio`

Output:
```
 minio.service - MinIO
     Loaded: loaded (/etc/systemd/system/minio.service; enabled; vendor preset: enabled)
     Active: inactive (dead)
     Assert: start assertion failed at Thu 2020-06-25 08:59:39 UTC; 1min 18s ago
             AssertFileIsExecutable=/usr/local/bin/minio was not met
       Docs: https://docs.min.io
```

**Comment by Christophvh** [comment]
I have solved it by manually installing mc, but that should be installed out of the box when enabling Minio as feature

```
wget https://dl.minio.io/client/mc/release/linux-amd64/mc
chmod +x mc
sudo mv mc /usr/local/bin
```

**Comment by svpernova09** [comment]
Resolved via v10.10.1
